keyform/models.py

```diff
@@ -89,3 +97,18 @@ class Meta:
 
     def __str__(self):
         return str(self.created_timestamp)
+
+class Contact(models.Model):
+    building = models.ManyToManyField(Building, blank=False)
+    name = models.CharField(max_length=30)
+    email = models.CharField(max_length=75)
```

Comment:
Did you think of using a [Email field](https://docs.djangoproject.com/en/1.11/ref/models/fields/#emailfield)